browser/brave_browser_main_parts.cc

```diff
@@ -158,6 +159,8 @@ void BraveBrowserMainParts::PreShutdown() {
 
 void BraveBrowserMainParts::PreProfileInit() {
   ChromeBrowserMainParts::PreProfileInit();
+  extensions::Extension::
```

Comment:
please add comment explaining what this is doing

Reply:
done